src/intro.md

```diff
@@ -2,14 +2,12 @@
 title: Introduction
 ---
 
-## The Grain Docs
+Welcome to the Grain docs! Here you'll find information about the language, the standard library, and officially supported tools. If you're brand new to Grain, we highly recommend starting with [the Grain guide](/docs), where you can have a tour of the language and make sure you're all set to write some code.
 
-These docs are divided into two sections—the **Standard Library** and **Language Constructs**.
+## Getting additional help
 
-### Standard Library
+If something isn't making sense or you have more questions, reach out in the `#support` channel on the [Grain Discord server](https://discord.com/invite/grain-lang). There are a bunch of friendly folks happy to help out!
 
-The Standard Library docs document all of the modules that comprise Grain's standard library, along with detailed information about the values that can be imported from them.
+## Improving the docs
 
-### Language Constructs
-
-The Language Constructs docs provide detailed information about the semantics of all of Grain's language features.
+Everything you see in the docs is completely open source and editable! If you think something could be explained more clearly or that some additional information could be helpful, we happily welcome pull requests. The "Edit on GitHub" buttons on each page will take you directly to where the changes would need to be made, and you can contribute without even leaving your browser!
```

Comment:
Kinda incorrect since stdlib docs are auto-generated now, but 🤷

Reply:
Going to update the links in another PR!